testutil/keeper/expectations.go

```diff
@@ -90,6 +91,63 @@ func GetMocksForStopConsumerChain(ctx sdk.Context, mocks *MockedKeepers) []*gomo
 	}
 }
 
+func GetMocksForHandleSlashPacket(ctx sdk.Context, mocks MockedKeepers,
+	expectedPacketData ccv.SlashPacketData, valToReturn stakingtypes.Validator) []*gomock.Call {
+
+	// These first two calls are always made.
+	calls := []*gomock.Call{
+
+		mocks.MockStakingKeeper.EXPECT().GetValidatorByConsAddr(
+			ctx, sdk.ConsAddress(expectedPacketData.Validator.Address)).Return(
+			valToReturn, true,
+		).Times(1),
+
+		mocks.MockSlashingKeeper.EXPECT().IsTombstoned(ctx,
+			sdk.ConsAddress(expectedPacketData.Validator.Address)).Return(false).Times(1),
+	}
+
+	// Different calls are made depending on the type infraction.
+	if expectedPacketData.Infraction == stakingtypes.Downtime {
+		calls = append(calls,
+			mocks.MockSlashingKeeper.EXPECT().SlashFractionDowntime(ctx).Return(sdk.NewDec(1)).Times(1),
+			mocks.MockSlashingKeeper.EXPECT().DowntimeJailDuration(ctx).Return(time.Hour).Times(1),
+		)
+
+	} else if expectedPacketData.Infraction == stakingtypes.DoubleSign {
+		calls = append(calls,
+			mocks.MockSlashingKeeper.EXPECT().SlashFractionDoubleSign(ctx).Return(sdk.NewDec(1)).Times(1),
```

Comment:
Nit: it might be nice to move the hardcoded NewDec(1)'s to the top of the function and caps name them.